### PR TITLE
Adds 2 companies on the list

### PR DESCRIPTION
**Pull request by ghaleroshan**
- [x] Adds JXT Global to the list
- [x] Adds Kayo Sports to the list

## Add/Update/Remove <CompanyName>

- [x] I have read the [contributing guidelines](https://github.com/geshan/au-companies-providing-work-visa-sponsorship/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/geshan/au-companies-providing-work-visa-sponsorship/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/geshan/au-companies-providing-work-visa-sponsorship/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [x] (OPTIONAL) In your pull request message, add additional context/communication on the visa sponsorship if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ A searchable sort-able and group-able version of the list of companies below is
 - [IRESS](https://www.iress.com/au/company/join-us/) | Melbourne CBD, Melbourne; Sydney CBD, Sydney | Reactjs, Python, JavaScript, TypeScript, AWS
 
 ## J - L
-- [Koala](https://www.linkedin.com/company/koala-mattress/jobs/) | Alexandria, Sydney | Go, PHP, Javascript, AWS
+- [JXT Global](https://stackoverflow.com/jobs/companies/jxt-global) | Sydney CBD, Sydney | .NET, C#, React, Angular, AWS
+- [Kayo Sports](https://stackoverflow.com/jobs/companies/kayo-sports) | Artamon, Sydney | HTML, Javascript, Css, Node.Js, React.Js, Express.js
```
Comment (geshan) [inline README.md]: HTML and CSS is not necessary here IMHO.

File: README.md [hunk 2]
```diff
@@ -37,7 +37,9 @@ A searchable sort-able and group-able version of the list of companies below is
 - [IRESS](https://www.iress.com/au/company/join-us/) | Melbourne CBD, Melbourne; Sydney CBD, Sydney | Reactjs, Python, JavaScript, TypeScript, AWS
 
 ## J - L
-- [Koala](https://www.linkedin.com/company/koala-mattress/jobs/) | Alexandria, Sydney | Go, PHP, Javascript, AWS
+- [JXT Global](https://stackoverflow.com/jobs/companies/jxt-global) | Sydney CBD, Sydney | .NET, C#, React, Angular, AWS
```
Comment (geshan) [inline README.md]: add links of the main site not StackOverflow.

File: README.md [hunk 3]
```diff
@@ -37,7 +37,9 @@ A searchable sort-able and group-able version of the list of companies below is
...
+- [Kayo Sports](https://stackoverflow.com/jobs/companies/kayo-sports) | Artamon, Sydney | HTML, Javascript, Css, Node.Js, React.Js, Express.js 
+- [Koala](https://www.linkedin.cohttps://stackoverflow.com/jobs/companies/kayo-sportsm/company/koala-mattress/jobs/) | Alexandria, Sydney | Go, PHP, Javascript, AWS
```
Comment (geshan) [inline README.md]: I guess you have messed this up by mistake.

Comment (ghaleroshan) [reply README.md]: oh shit. Yep. Will fix it

**Pull request merged**